signer/src/main/java/com/uport/sdk/signer/UportHDSigner.kt

```diff
@@ -93,8 +93,8 @@ class UportHDSigner : UportSigner() {
 
                 return@storeEncryptedPayload callback(null, address, publicKeyString)
             }
-        } catch (ex: Exception) {
-            return callback(ex, "", "")
+        } catch (seedImprotError: Exception) {
```

Comment:
The error caught here is a generic `Exception`, is the aim to specifically catch a `SeedImportationException`?

Reply:
The idea is not to handle specific exceptions but to funnel them along into the `callback` method.
The name is there just to differentiate between the multiple levels where the callback can be called from.